monitoring/uss_qualifier/scd/simulator/utils.py

```diff
@@ -0,0 +1,104 @@
+from pathlib import Path
+from typing import List, Union, Literal
+from geojson import Feature
+import enum
+from shapely.geometry import LineString, Polygon
+from monitoring.monitorlib.typing import ImplicitDict, StringBasedDateTime
+from monitoring.monitorlib.scd import Altitude, Volume4D,  OperationalIntentState
+from pathlib import Path
+
+class OutputSubDirectories(ImplicitDict):
+    ''' A class to hold information about output directories that will be generated when writing the files to disk. '''
+    injection_payloads: Path 
+    scd_rules: Path
+
+class InjectionStatus(str, enum.Enum):
+    ''' A enum to hold results of flight processing as defined by the SCD test API '''
+    Planned = 'Planned'
+    Rejected = 'Rejected'
+    ConflictWithFlight = 'ConflictWithFlight'
+    Failed = 'Failed'
+
+## Definitions around operational intent data that needs to be submitted to the test injection interface
```

Comment:
We're missing the definition of `InjectFlightRequest` here, which is probably the most important definition for the next phase where we make sure we can supply a valid instance of that type to invoke the injection API.

Reply:
OK I have now added this to the file.